pkg/pidfile/pidfile_darwin.go

```diff
@@ -0,0 +1,26 @@
+// +build darwin
+
+package pidfile
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+)
+
+func processExists(pid int) bool {
+	// On Unix systems, FindProcess always succeeds.
+	// It returns a Process for the given pid, regardless of whether the process exists.
+	process, err := os.FindProcess(pid)
```

Comment:
Is this actually needed? According to the docs, it's basically a no-op; https://golang.org/src/os/exec_unix.go?h=findProcess#L70

It instances an object, with the `pid` you're passing: https://golang.org/src/os/exec.go?h=newProcess#L20